### ISSUE TITLE
Weird jackson dependency missing issue

### ISSUE THREAD

**Issue by cvakiitho** [sylvainlaurent/yaml-json-validator-maven-plugin#25]
Hi, I'm not really sure this is this projects issue, but I'm a bit lost, so maybe someone saw simillar problem.

```
[ERROR] Failed to execute goal com.github.sylvainlaurent.maven:yaml-json-validator-maven-plugin:1.0.4:validate (test-schema) on project  <>: Execution test-schema of goal com.github.sylvainlaurent.maven:yaml-json-validator-maven-plugin:1.0.4:validate failed: A required class was missing while executing com.github.sylvainlaurent.maven:yaml-json-validator-maven-plugin:1.0.4:validate: com/fasterxml/jackson/databind/ObjectMapper
```

This error started happening yesterday inside jenkins builds, but it works locally without problem, 
I checked dependency tree, and jackson 2.9.9 is there, no idea what's going on, and why it started happening without any external change.

**Comment by cvakiitho** [comment]
btw this is fixed by sylvainlaurent/yaml-json-validator-maven-plugin/pull/22

**Comment by sylvainlaurent** [comment]
thank you for your help, I merged your PR #22
